beets/ui/commands.py

```diff
@@ -496,6 +496,9 @@ def _summary_judgment(rec):
             'asis': importer.action.ASIS,
             'ask': None,
         })
+        # prompt the user if timid is enabled
+        if config['import']['timid'] and action == importer.action.ASIS:
```

Comment:
Is `importer.action.ASIS` the proper tool to use here? The as_choice function from beetbox/confuse complicates things.

Reply:
Yep, looks right to me.